quickevent/app/plugins/Runs/src/runsplugin.cpp

```diff
@@ -1083,6 +1084,20 @@ qf::core::utils::TreeTable RunsPlugin::startListClassesTable(const QString &wher
 		.orderBy("runs.startTimeMs, competitors.lastName");
 	for(int i=0; i<tt.rowCount(); i++) {
 		qf::core::utils::TreeTableRow tt_row = tt.row(i);
+		// add number of controls
+		QString query_str = "SELECT COUNT(*) FROM codes,coursecodes"
```

Comment:
```suggestion
		QString query_str = "SELECT COUNT(*) FROM codes,coursecodes"
					" WHERE codes.id=coursecodes.codeId"
					" AND codes.code>=" QF_IARG(quickevent::core::CodeDef::PUNCH_CODE_MIN)
					" AND codes.code<=" QF_IARG(quickevent::core::CodeDef::PUNCH_CODE_MAX)
					" AND coursecodes.courseId=" QF_IARG(tt_row.value("courses.id").toInt());
```

Reply:
jeste je potreba `courses.id` pridat do selectu nahore
`.select2("courses", "length, climb")`
jinak je to perfektni :)

Reply:
díky, to tomu chybělo ;)